pkg/pubsub/i_pubsub.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The OpenPitrix Authors. All rights reserved.
```

Comment:
Not to use abbreviation will be better，i means a lot.

Reply:
update